src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -616,4 +616,7 @@
   <data name="PX1036FKFix" xml:space="preserve">
     <value>Place all foreign keys in a public static class named "FK"</value>
   </data>
+  <data name="PX1076Title" xml:space="preserve">
+    <value>Calls to Acumatica internal API marked with PXInternalUseOnlyAttribute are forbidden</value>
```

Comment:
@EAndrosova  please review diagnostic warning message

Reply:
I think we need to elaborate further on this. This string is too strict (not polite enough) for external developers. I think it should be something like `This code calls Acumatica internal API marked with PXInternalUseOnlyAttribute that is not intended for public use`. @EAndrosova what do you think?

Reply:
Maybe we should add something like "we give no guarantee that it won't be changed in the future" or "is subject to changes in the future"?

Reply:
I think the details about possible consequences can be explained in the diagnostic's description. It is too long and cumbersome to fit it to the title itself.

Reply:
@dropsonic  I like your suggestion, I will replace the string as part of the review